packages/lab/src/color-chooser/HexInput.tsx

```diff
@@ -19,6 +22,13 @@ export const HexInput = ({
   disableAlphaChooser,
   onSubmit,
 }: HexInputProps): JSX.Element => {
+  const { window: targetWindow } = useWindow();
+  useComponentCssInjection({
+    testId: "salt-color-picker-hex-input",
```

Comment:
please make it consistent with value in 'packages/lab/src/color-chooser/AlphaInputField.tsx'